daniela-ticona/lesson-3/GuessMovie/src/Main.java

```diff
@@ -0,0 +1,9 @@
+public class Main {
+    public static void main(String[] args) throws Exception {
+
+        new MovieClass();
```

Comment:
this works but It is not a good practice to do run the program in the constructor, you should get the movie to guess and in method and call this and FindMatch here in the main method.